specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/workbookTemplates_API.json

```diff
@@ -0,0 +1,467 @@
+      }
+    }
+  },
+  "definitions": {
+    "WorkbookTemplateResource": {
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "WorkbookTemplatesListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/WorkbookTemplate"
+          },
+          "description": "An array of workbook templates."
+        }
+      },
+      "description": "WorkbookTemplate list result."
+    },
+    "WorkbookTemplate": {
+      "description": "An Application Insights workbook template definition.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/WorkbookTemplateResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a workbook template for an Azure resource.",
+          "$ref": "#/definitions/WorkbookTemplateProperties"
+        }
+      }
+    },
+    "WorkbookTemplateProperties": {
+      "description": "Properties that contain a workbook template.",
+      "required": [
+        "id",
+        "templateData",
+        "galleries"
+      ],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Id of the workbook template. This is GUID value."
+        },
+        "priority": {
+          "type": "integer",
+          "description": "Priority of the template. Determines which template to open when a workbook gallery is opened in viewer mode."
+        },
+        "author": {
+          "type": "string",
+          "description": "Information about the author of the workbook template."
+        },
+        "templateData": {
+          "type": "object",
+          "description": "Valid JSON object containing workbook template payload."
+        },
+        "galleries": {
+          "type": "array",
+          "description": "Workbook galleries supported by the template.",
+          "items": {
+            "$ref": "#/definitions/WorkbookTemplateGallery"
+          }
+        },
+        "localized": {
```

Comment:
What is this?  Why not return the overall resource in the locale of the calling user instead of having a collection of localized content?

Reply:
When authors create a template, they can provide the localized version of their template in different locales as well. For now we are returning the complete payload for all gets but the next iteration will add optimizations around how we store, query and return localized templates.